VMEncryption/main/patch/centosPatching.py

```diff
@@ -74,39 +74,6 @@ def __init__(self, logger, distro_info):
             self.umount_path = '/usr/bin/umount'
 
     def install_extras(self):
-        epel_packages_installed = False
-        attempt = 0
-
-        while not epel_packages_installed:
-            attempt += 1
-            self.logger.log("Attempt #{0} to locate EPEL packages".format(attempt))
-            if self.distro_info[1].startswith("6."):
-                if self.command_executor.Execute("rpm -q ntfs-3g python-pip"):
```

Comment:
>python-pip [](start = 65, length = 10)

Is python-pip no longer required?

Reply:
python-pip was used to install adal which is no longer required by the single pass extension